spec/models/post_mover_spec.rb

```diff
@@ -664,6 +672,18 @@ def create_topic_user(user, opts = {})
             expect(timer).to be_nil
           end
 
+          it "ignores moderator posts and closes the topic if all regular posts were moved" do
+            add_moderator_post_to topic, Post.types[:moderator_action]
+            add_moderator_post_to topic, Post.types[:small_action]
+
+            posts_to_move = [p1.id, p2.id, p3.id, p4.id]
+            moved_to = topic.move_posts(user, posts_to_move, destination_topic_id: destination_topic.id)
+            expect(moved_to).to be_present
```

Comment:
Generally, I think we should avoid generic assertions like `be_present` because it makes the test too generic and doesn't make it clear what object are we expecting to be present. Perhaps something like the following?

```
expect(moved_to).to eq(topic)
```

Reply:
>  I think we should avoid generic assertions like be_present

I agree 👍 

Moreover, I think in this specific test case no need to check if the topic was returned, so I've removed this check.

What we're checking here is that _"... it closes the topic if all regular posts were moved"_. Another assert do that: `expect(topic).to be_closed`.